tools/build-companion.sh

```diff
@@ -20,42 +20,45 @@ if [ "$3" == "" ]; then
    exit 1
 fi
 
+if [ "$JOBS" == "" ]; then
+    JOBS=2
+fi
```

Comment:
I would prefer if JOBS was an option to the script like in https://github.com/opentx/opentx/blob/next/tools/commit-tests.sh, but then you have to take care of the other two arguments also. Nothing critical if it stays the way it is here.